hamocc/meson.build

```diff
@@ -1,19 +1,16 @@
 sources += files(
-  'accfields.F90',
-  'aufr_bgc.F90',
-  'aufw_bgc.F90',
-  'mo_ini_fields.F90',
-  'carchm.F90',
-  'carchm_kequi.F90',
-  'carchm_solve.F90',
-  'carchm_solve_DICsat.F90',
-  'cyano.F90',
-  'dipowa.F90',
-  'get_cfc.F90',
-  'hamocc4bcm.F90',
-  'hamocc_init.F90',
-  'hamocc_step.F90',
-  'inventory_bgc.F90',
+  'mo_accfields.F90',
+  'mo_aufr_bgc.F90',
+  'mo_aufw_bgc.F90',
+  'mo_mo_ini_fields.F90',
```

Comment:
Hi @mvertens , why `mo_mo_ini_fields`? - one `mo_` is enough :-) - might be the cause for the failing tests on github.

Reply:
Sorry for the typo. I'm not using meson build - but the NorESM CIME build. I'll fix this in my upcoming push.
Thanks for catching this.